e2e/tools/validator/src/validator/cli/__init__.py

```diff
@@ -77,23 +78,28 @@ def stress(cfg: Validator, script_path: str):
     # mape = mean_absolute_percentage_error(expected_data, actual_data)
     
     test_cases = Cases(vm = cfg.metal.vm, prom = cfg.prometheus, query_path = cfg.query_path)
-    metrics_validator = MetricsValidator(cfg.prometheus)
+    metrics_validator = MetricsValidator(cfg.prometheus, cfg.remote_prometheus)
     test_case_result = test_cases.load_test_cases()
     click.secho("Validation results during stress test:")
-    click.secho("Validation results during stress test:")
+    # run git describe command and get the output as the report name
+    tag = ""
+    git_describe = subprocess.run(["git", "describe", "--tag"], stdout=subprocess.PIPE)
+    if git_describe.stdout:
+        tag = git_describe.stdout.decode().strip()
```

Comment:
Could you please submit another PR for this change since supporting remote prometheus shouldn't require `git describe`